Makefile

```diff
@@ -37,20 +61,23 @@ image: build
 
 # Deploys Vault dev server and a locally built Agent Injector.
 # Run multiple times to deploy new builds of the injector.
+VAULT_HELM_POST_INSTALL_ARGS ?=
+ifndef TEST_WITHOUT_VAULT_TLS
+	VAULT_HELM_POST_INSTALL_ARGS = "--set=injector.extraEnvironmentVars.AGENT_INJECT_VAULT_CACERT_BYTES=$$(kubectl exec vault-0 -- sh -c 'cat /tmp/vault-ca.pem | base64 -w0')"
+endif
 deploy:
-	kind load docker-image hashicorp/vault-k8s:$(VERSION)
-	helm upgrade --install vault vault $(VAULT_HELM_FLAGS) \
+	kind load docker-image --name $(KIND_CLUSTER_NAME) $(IMAGE_TAG)
+	helm upgrade --install vault vault $(VAULT_HELM_DEFAULT_ARGS) \
 		--set "injector.enabled=false"
 	kubectl delete pod -l "app.kubernetes.io/instance=vault"
 	kubectl wait --for=condition=Ready --timeout=5m pod -l "app.kubernetes.io/instance=vault"
-	helm upgrade --install vault vault $(VAULT_HELM_FLAGS) \
-		--set "injector.extraEnvironmentVars.AGENT_INJECT_VAULT_CACERT_BYTES=$$(kubectl exec vault-0 -- cat /tmp/vault-ca.pem | base64)"
+	helm upgrade --install vault vault $(VAULT_HELM_DEFAULT_ARGS) $(VAULT_HELM_POST_INSTALL_ARGS)
 
 # Populates the Vault dev server with a secret, configures kubernetes auth, and
 # deploys an nginx pod with annotations to have the secret injected.
 exercise:
 	kubectl exec vault-0 -- vault kv put secret/test-app hello=world
-	kubectl exec vault-0 -- vault auth enable kubernetes || true
+	kubectl exec vault-0 -- vault auth enable kubernetes 2> /dev/null || true
```

Comment:
Why is this hiding stderr? Seems like that could make it harder to debug?

Reply:
To avoid red herrings when debugging the build. I can make this a warning instead.

Reply:
Fixed as of the latest commit. Enabling the k8s auth backend is now idempotent.